### PR TITLE
Feature/bb 3179

### PR DESCRIPTION
**Pull request by aethemba**

### CODE REVIEW

File: bluebottle/bb_projects/serializers.py [hunk 1]
```diff
@@ -1,41 +1,51 @@
...
         """ Saving the story text """
-        #Convert &gt; and &lt; back to HTML tags so Beautiful Soup can clean unwanted tags.
-        #Script tags are sent by redactor as "&lt;;script&gt;;", Iframe tags have just one semicolon.
-        data = data.replace("&lt;;", "<").replace("&gt;;", ">").replace("&lt;", "<").replace("&gt;", ">")
+        # Convert &gt; and &lt; back to HTML tags so Beautiful
+        # Soup can clean unwanted tags.
+        # Script tags are sent by redactor as "&lt;;script&gt;;",
+        # Iframe tags have just one semicolon.
+        data = data.replace("&lt;;", "<").replace(
+            "&gt;;", ">").replace("&lt;", "<").replace("&gt;", ">")
```
Comment (gannetson) [inline bluebottle/bb_projects/serializers.py]: For readability maybe rewrite thinks like this as
```
data = data.replace("&lt;;", "<")
data = data.replace("&gt;;", ">")
data = data.replace("&lt;", "<")
data = data.replace("&gt;", ">")
```
or even smart in a pattern like
```
repls = {'hello' : 'goodbye', 'world' : 'earth'}
s = 'hello, world'
reduce(lambda a, kv: a.replace(*kv), repls.iteritems(), s)
```

File: bluebottle/projects/migrations/0014_bank_details.py [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+    depends_on = (
+        ('organizations', '0002_auto__add_field_organizationdocument_created__add_field_organizationdo'),
+    )
+
+    def forwards(self, orm):
+        # Note: Don't use "from appname.models import ModelName".
+        # Use orm.ModelName to refer to models in this application,
+        # and orm['appname.ModelName'] for models in other applications.
+        print "GKENSGKENGKFLNLNWLFNLKESNFGKLENLKEW************"
```
Comment (gannetson) [inline bluebottle/projects/migrations/0014_bank_details.py]: Hear! Hear!

File: bluebottle/projects/migrations/0014_bank_details.py [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+                project.account_bank_name = organization.account_bank_name
+                project.account_bank_address = organization.account_bank_address
+                project.account_bank_postal_code = organization.account_bank_postal_code
+                project.account_bank_city = organization.account_bank_city
+                project.account_bank_country = organization.account_bank_country
+                project.account_other = organization.account_other
+
+                project.save()
+            print "---"
+        print '*****'
```
Comment (gannetson) [inline bluebottle/projects/migrations/0014_bank_details.py]: It's a free beer for every dev team member every 'print' or 'console.log' found in a PR. 
Two beers if a reviewer doesn't find them...

**Comment by gannetson** [comment]
Not entirely sure why bank details should be on the project. Did you confirm this with Malou or Joris?
For now I'll still merge this in so we can move forward.

**Comment by aethemba** [comment]
@gannetson , yeah this is confirmed with Joris and @rollick. Not with Malou though

**Pull request merged**